### PR TITLE
[#12048] SeedDb code for student entities

### PR DESCRIPTION
**Pull request by marquestye**
Part of #12048 

**Outline of Solution**
- Added method to seed students in SeedDb.java
    - Currently adds only one student per course, since the number of courses and accounts seeded are the same

### CODE REVIEW

**Review by NicolasCwy — APPROVED** [review]
LGTM, after #12986 is merged update this branch and use any shared constants before we merge it

File: src/client/java/teammates/client/scripts/sql/SeedDb.java
Comment (mingyuanc) [inline src/client/java/teammates/client/scripts/sql/SeedDb.java:139]: Should seedStudents be in the try block? such that if there is an exception then seedStudents will not run

**Review by mingyuanc — CHANGES_REQUESTED** [review]

File: src/client/java/teammates/client/scripts/sql/SeedDb.java
Comment (NicolasCwy) [reply src/client/java/teammates/client/scripts/sql/SeedDb.java:139]: Makes sense, could abstract course creation to a function and put both of the methods in the try block

**Review by FergusMok — APPROVED** [review]
Great job, LGTM

**Pull request closed without merging**
